Python/optimizer_analysis.c

```diff
@@ -571,6 +698,8 @@ _Py_uop_analyze_and_optimize(
     assert(err == 1);
 
     remove_unneeded_uops(buffer, buffer_size);
+    inline_simple_frames(buffer, buffer_size);
+    // remove_unneeded_uops(buffer, buffer_size);
```

Comment:
Interesting. Were you planning on calling this twice, or just swapping the two calls around?